games/CartPole-v0.jl

```diff
@@ -0,0 +1,94 @@
+import Reinforce
+using Reinforce:CartPoleV0, actions, reset!, finished, step!
+using Flux, StatsBase, Plots
+
+gr()
+
+#-------------Initialise game environment--------------#
+env = CartPoleV0()
+
+#---------------Parameters--------------#
+EPISODES = 3000
+STATE_SIZE = length(env.state)
+ACTION_SIZE = length(actions(env, env.state))
+
+MEM_SIZE = 2000
+BATCH_SIZE = 32
+
+γ = 0.95                # discount rate
+η = 0.001               # learning rate
+
+ϵ = 1.0                 # exploration rate
+ϵ_min = 0.01            # exploration minimum
+ϵ_decay = 0.995         # exploration decay
+
+memory = []             # used to remember past results
```

Comment:
Since memory functions like a queue of fixed size, [CircularBuffer](http://juliacollections.github.io/DataStructures.jl/latest/circ_buffer.html#CircularBuffer-1) from DataStructures.jl can be used.

Reply:
I will get to it.